app/src/main/java/com/example/moviews/data/remote/GetJsonFromUrl.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.moviews.data.remote
+
+import android.os.AsyncTask
+import com.example.moviews.data.OnLoadDataCallback
+import org.json.JSONObject
+
+class GetJsonFromUrl<T> constructor(
+    private val callback: OnLoadDataCallback<T>,
+    private val keyEntity: String
+) : AsyncTask<String?, Unit, String?>() {
```

Comment:
Sai thứ tự truyền vào param, check conventions 
```suggestion
class GetJsonFromUrl<T>(
    private val callback: OnLoadDataCallback<T>,
    private val keyEntity: String
) : AsyncTask<String?, Unit, String?>() {
```